lighthouse-core/test/audits/seo/font-size-test.js

```diff
@@ -238,7 +238,7 @@ describe('SEO: Font size audit', () => {
     expect(auditResult.notApplicable).toBe(true);
   });
 
-  describe('attributes source location', () => {
+  describe('attributes style source', () => {
```

Comment:
`attributes style source` are all nouns so.... wb `attributes source of style` ? still all nouns but BAM a preposition too

Reply:
haha love it!